README.md

```diff
@@ -456,6 +456,21 @@ If you need to build lib/lexer.wat (optional) you must first install
 [wabt](https://github.com/WebAssembly/wabt) as a sibling folder to this
 project. The wat file is then build by running `make lib/lexer.wat`
 
+### Creating a Release
+These are the steps to create and publish a release. You will need docker
+installed as well as having installed [wabt](https://github.com/WebAssembly/wabt)
+as outlined above:
+
+- [ ] Update the package.json version, and run a full build and test
```

Comment:
```suggestion
```suggestion
- [ ] Figure out if the release should be semver patch, minor or major based on the changes since
      the last release and determine the new version.
```suggestion
- [ ] Figure out if the release should be semver patch, minor or major based on the changes since
      the last release and determine the new version.
- [ ] Commit and tag the changes, pushing up to main and the tag
``` 
``` 
```